src/nifake/metadata/attributes.py

```diff
@@ -1,5 +1,5 @@
 # -*- coding: utf-8 -*-
-# This file is generated from NI-FAKE API metadata version 21.0.0d11
+# This file is generated from NI-FAKE API metadata version 22.5.0d9999
```

Comment:
I'll update this to an official export once the internal PR gets approved and merged.

Reply:
I will split this PR into smaller PRs and this PR will now updata the NI-FAKE metadata to 22.5.0d9